### ISSUE TITLE
aria-placeholder is empty when placeholder is set

### ISSUE THREAD

**Issue by EvgenyOrekhov** [remirror/remirror#2063]
### Summary

`aria-placeholder` is empty even though `placeholder` is set

### Steps to reproduce

Set `placeholder="Test"`

### Expected results

`aria-placeholder` should have value `"Test"`

### Actual results

`aria-placeholder` is empty

### Screenshot(s)

<img width="1390" alt="image" src="https://github.com/remirror/remirror/assets/8045060/005ea99a-c6f0-45aa-b57b-c882a65b3aa2">
